src/components/SkyBrowser/InfoBoxSkybrowser.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { times } from 'lodash';
+import MaterialIcon from '../common/MaterialIcon/MaterialIcon';
+import Button from '../common/Input/Button/Button';
+import TooltipSkybrowser from './TooltipSkybrowser';
+import styles from './TooltipSkybrowser.scss';
+import esaSkyLogo from './ESASKY.png';
+
+class InfoBoxSkybrowser extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { showPopup: false, tooltipActive: false};
+    this.setRef = this.setRef.bind(this);
```

Comment:
I would put a blank line between setting the state and the binds, to visually distinguish them and make it easier to see where the state is being defined